jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,16 @@
+lib = library(identifier: 'jenkins@1.5.3', retriever: modernSCM([
```

Comment:
Maybe use recent version?
```suggestion
lib = library(identifier: 'jenkins@4.2.2', retriever: modernSCM([
```

Reply:
Should I update this for JDBC driver too?